include/mem.h

```diff
@@ -56,7 +54,7 @@ MemHandle MEM_NextHandle(MemHandle handle);
 MemHandle MEM_NextHandleAt(MemHandle handle,Bitu where);
 
 // Read and write single-byte values
-static INLINE uint8_t host_readb(const uint8_t *var)
+constexpr uint8_t host_readb(const uint8_t *var)
 {
 	return *var;
 }
```

Comment:
(Attaching comment to this function, but it applies to all `host_*` functions in this file).

I looked again at the code and compared to the version existing upstream right now. All `host_read*` and `host_write*` functions diverged very much from the versions available upstream.

To make maintenance of our changes easier, I think we should move all `host_*` functions to a separate header. This will make it easier to maintain our changed implementation, and easier to resolve future conflicts with upstream `mem.h`. I suggest placing them in a header `mem_host.h`.

Functions `var_write` should stay in this file as they are.

I think this new header should be introduced in this PR - this way in future PR if we'll need to add new functions like `host_readw_at`, the implementation will be much clearer and won't touch `mem.h` any more.

Reply:
Nice suggestion; moved them over to `mem_host.h`.